src/components/common/BottomSheet/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/css'
+import { MouseEventHandler, ReactNode } from 'react'
+
+type BottomSheetProps = {
+  isOpen: boolean
+  onClose: () => void
+  children: ReactNode
+}
+
+export const BottomSheet = ({
+  isOpen,
+  onClose,
+  children
+}: BottomSheetProps) => {
```

Comment:
```suggestion
type BottomSheetProps = {
  isOpen?: boolean
  onClose?: () => void
}

export const BottomSheet = ({
  isOpen = false,
  onClose,
}: PropsWithChildren<BottomSheetProps>) => {
```

- 기본값 활용하기
- PropsWithChildren 활용하기

Reply:
[72d3ef5](https://github.com/next-step/react-payments/pull/134/commits/72d3ef57e47028ff52a29093d50ac6c3209ebfb9)에서 반영하였습니다! 감사합니다!